tenacity/_utils.py

```diff
@@ -26,7 +26,7 @@
     MAX_WAIT = sys.maxint / 2
 except AttributeError:
     MAX_WAIT = 1073741823
-
+MIN_WAIT = 0
```

Comment:
Just go ahead and use 0 directly. The `MAX_WAIT` here is just because you can't be sure what the max int might be.